change/@fluentui-react-next-2020-09-18-16-25-30-pickers-fix.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "Floating Pickers: Make compliant",
```

Probably good to specify that compliant refers to strict mode (there are lots of possible types of "compliant")
```suggestion
  "comment": "Floating Pickers: Make strict mode compliant",
```